### PR TITLE
[SHEPHERD] Add linter for python files

### PR DESCRIPTION
**Pull request by brandonxxlee**
Add pylint to lint all python files in the shepherd directory

### CODE REVIEW

**Review by jonathan-j-lee — DISMISSED** [review]
LGTM

File: shepherd/Shepherd.py [hunk 1]
```diff
@@ -17,16 +18,14 @@ def __init__(self, matchNumber):
         self.alliances= {'blue': Alliance(teams[0], teams[1]), 'gold': Alliance(teams[2], teams[3])}
         self.currentStage = 0
 
-        #The follwing are elements that are configured specifically for the 2018 year game - Solar Scramble
+        #The follwing are elements that are configured
```
Comment (mc10) [inline shepherd/Shepherd.py]: s/follwing/following

File: shepherd/Shepherd.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def __init__(self, matchNumber):
         self.alliances= {'blue': Alliance(teams[0], teams[1]), 'gold': Alliance(teams[2], teams[3])}
         self.currentStage = 0
 
-        #The follwing are elements that are configured 
+        #The follwing are elements that are configured
```
Comment (kevinji) [inline shepherd/Shepherd.py]: s/follwing/following

Comment (brandonxxlee) [reply shepherd/Shepherd.py]: We'll fix this later when this file is actually worked on (I hope we remember)

**Review by kevinji — APPROVED** [review]

**Review by JHGZhou — APPROVED** [review]
Tabs are still better imo

**Pull request merged**